regclient/regclient
# Copy many images to ocidir

### Fixes issue

Relates to #710 to help copy many images into an ocidir.

### Describe the change

Add support for ocidir [URI fragment](https://datatracker.ietf.org/doc/html/rfc3986#section-3.5) to specify the image in index.

The index gains the `io.containerd.image.name` annotation for full image name.

### How to verify it

```
regctl image copy alpine "ocidir://t#test"
regctl image copy alpine:edge "ocidir://t#edge:latest"
```

This should result in two images in the index.

As this is currently, this does not work as intended. It still matches by the tag rather than by the image name. Therefore I keep this as draft.

### Changelog text



### Please verify and check that the pull request fulfills the following requirements
